rpc/types.go

```diff
@@ -86,6 +87,9 @@ func (bn *BlockNumber) UnmarshalJSON(data []byte) error {
 	case "pending":
 		*bn = PendingBlockNumber
 		return nil
+	case "justified":
```

Comment:
the doc need to be updated as well, currently: `"latest", "earliest" or "pending" as string arguments`